README.md

```diff
@@ -9,17 +8,45 @@ Monitore your websites availability, http status code (current and history), cer
 * [Prometheus blackbox exporter](https://github.com/prometheus/blackbox_exporter)
 * [Grafana](https://github.com/grafana/grafana)
 
-## Dependencies
+## Deploying
 
-* [docker](https://docs.docker.com/install/)
-* [docker-composer](https://docs.docker.com/compose/install/)
+- Kubernetes
 
-## Usage
+  - Dependencies
 
-* `git clone https://github.com/jwcesign/website-monitoring.git && cd website-monitoring`
-* Edit `config/prometheus/targets.yml` (see targets.yml.example) or use `./gen_target.sh website-1.tld website-2.tld ...`
-* Create and start containers `docker-compose up -d`
-* [Visualize dashboards](http://localhost:3000/)
+    * cluster
+      - [kubeadm](https://kubernetes.io/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+      - [minikube](https://minikube.sigs.k8s.io/docs/start/)
+      - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+    * [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+  - Usage
+
+    * `git clone https://github.com/cloudpilot-ai/website-monitoring.git && cd website-monitoring`
+    * Edit `sample/prometheus-targets-cm.yaml` (see targets.yml.example) modify or add `baidu.com` as your desired service address
+    * Create Namespace `kubectl create namespace prometheus`
+    * Starting services `kubectl apply -n prometheus -f sample`
+    * Visualize dashboards http://10.105.212.73:3000/ `kubectl -n prometheus get svc blackbox-exporter` get request address
```

Comment:
I guess using command `kubectl port-forward xxx` is a better choice? then visit `http://locaohost:3000`.

Reply:
ok, add a temporary solution